tools/slo-generator/.pylintrc

```diff
@@ -0,0 +1,570 @@
+[MASTER]
```

Comment:
can we add this to gitignore?

Reply:
sorry, having trouble understanding. What exactly should be added to gitignore ?

Reply:
the pylintrc file. is there a reason we need it here?

Reply:
there is a `make` action that runs tests using Pylint, taking this config as input.